jsk_unitree_robot/cross/install.sh

```diff
@@ -142,8 +145,13 @@ if [[ "${TARGET_DIRECTORY}" == "User" ]]; then
     sshpass -p 123 scp ${TARGET_MACHINE}_${TARGET_DIRECTORY}/src/jsk_robot/jsk_unitree_robot/jsk_unitree_startup/scripts/publish_human_pose.diff unitree@${cuda_ip}:/tmp/publish_human_pose.diff
     sshpass -p 123 ssh -t unitree@${cuda_ip} bash -c 'ls; OUT="$(patch -p0 --backup --forward /home/unitree/Unitree/autostart/imageai/mLComSystemFrame/pyScripts/live_human_pose.py < /tmp/publish_human_pose.diff | tee /dev/tty)" || echo "${OUT}" | grep "Skipping patch" -q || (echo "$OUT" && false);'
 
-    if [[ ${TYPE} == "Air" ]] ; then
-        sshpass -p 123 ssh -t unitree@${cuda_ip} "sed -i 's/192.168.123.15/192.168.123.13/g' /home/unitree/Unitree/autostart/imageai/mLComSystemFrame/pyScripts/live_human_pose.py"
+    # launch live_human_pose.py on jetson nano (192.168.123.13)
```

Comment:
why you changed to start publish_human_pose from 192.168.123.15 to 192.168.123.13? To have same setting for both Air/Pro?

Reply:
Sorry. This change is no longer necessary.
This was to publish the image with ROS.

Instead, I used mqtt to publish.
https://github.com/k-okada/jsk_robot/pull/69/commits/9b2b375d5dab45acad4f4a914dcf37d1d44c2070